api/client/trust.go

```diff
@@ -156,7 +156,7 @@ func (cli *DockerCli) getNotaryRepository(repoInfo *registry.RepositoryInfo, aut
 	}
 
 	creds := simpleCredentialStore{auth: authConfig}
-	tokenHandler := auth.NewTokenHandler(authTransport, creds, repoInfo.CanonicalName, "push", "pull")
+	tokenHandler := auth.NewTokenHandler(authTransport, creds, repoInfo.CanonicalName, []string{"push", "pull"})
```

Comment:
No need to wrap in a `[]string{}`, the last argument to `auth.NewTokenHandler` is `actions ...string`

https://github.com/docker/distribution/blob/master/registry/client/auth/session.go#L113